x-pack/plugins/observability_solution/observability_shared/common/locators/entity_inventory/entity_inventory_locator.ts

```diff
@@ -16,8 +16,8 @@ export class EntitiesInventoryLocatorDefinition implements LocatorDefinition<Ser
 
   public readonly getLocation = async () => {
     return {
-      app: 'observability',
-      path: `/inventory`,
+      app: 'inventory',
+      path: `/`,
```

Comment:
This locator could accept params to build the necessary querystring eg: [service_overview_locator](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/observability_shared/common/locators/apm/service_overview_locator.ts)

Reply:
I won't change that for now... We're going to remove the control groups which will change the query params again.

Reply:
for the initial loading of the inventory, coming from APM service overview we don't necessarily need any query params, do we?

Reply:
No